js/utils/datoUtils.js

```diff
@@ -32,15 +32,7 @@ export const datoMedKlokkeslett = (dato) => {
 };
 
 export const newDate = () => {
-  const now = new Date();
-  return new Date(
-    now.getUTCFullYear(),
-    now.getUTCMonth(),
-    now.getUTCDate(),
-    now.getHours(),
-    now.getUTCMinutes(),
-    now.getUTCSeconds()
-  );
+  return new Date();
```

Comment:
trenger man en util for new Date?

Reply:
Hehe, nei. Dette er en gammel funksjon som vi ikke tørr å slette men som basically genererte et Date object